examples/profiling_voice.py

```diff
@@ -0,0 +1,28 @@
+# +
+import torch
+import torch.tensor as T
+
+from torchsynth.globals import SynthGlobals
+from torchsynth.synth import Voice
+
+# %load_ext autoreload
+# %autoreload 2
+# -
+
+if torch.cuda.is_available():
+    device = 'cuda'
+else:
+    device = 'cpu'
+
+synthglobals64 = SynthGlobals(
+    batch_size=T(16), sample_rate=T(44100), buffer_size=T(4 * 44100)
```

Comment:
Just curious if we should profile:
a) what happens if the SynthGlobals tensors are on the device
b) Other keep them as ints for simplificity?

Reply:
ya I'll make an issue for the profiling stuff for each module -- Or a project perhaps and keep things organized for each module

Reply:
Made a project with issues for profiling individual modules